.github/workflows/pytest-gpu.yml

```diff
@@ -91,6 +95,9 @@ jobs:
           DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test tests/test_gpu_openmp.py
           DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test examples/seismic/elastic/elastic_example.py
           DEVITO_ARCH=clang DEVITO_PLATFORM=nvidiaX py.test examples/seismic/viscoelastic/viscoelastic_example.py
+          unset DEVITO_PLATFORM;
```

Comment:
I don't like it

I think the env vars should now be centralised in the action setup, so that you don't have to set/unset anything at this stage

Reply:
I agree. I will try to fix this together with the general env variables hiding.

Reply:
what happens if you just drop the `unset` ?